practice.md

```diff
@@ -21,15 +21,16 @@ To the extent 18F has the rights to do so, 18F will normalize the copyright stat
 
 ### Contributing back to outside projects
 
-18F staff are encouraged to contribute back any modifications or improvements they make to open source software projects outside 18F -- whether government or non-government -- in the course of their work. When 18F staff begin modifications to outside work, they should plan with eventual upstream contribution in mind.
+18F staff are encouraged to seek existing, open source solutions -- whether government or non-government -- before writing custom tools. When existing libraries need to be modified or improved, 18F staff should make the modifications with eventual upstream contribution in mind.
+
+In practice, this generally involves forking the relevant repository to the 18F organization within GitHub, creating a new branch with the modifications, and sending a pull request to upstream from the 18F fork. Unlike our own projects, there is no need for internal code review in this scenario (though it doesn't hurt).
```

Comment:
I, too, have seen both styles in use; the goal here is to prefer the 18F org. We have a records retention policy that promises we keep all of our code around in Github, which isn't as easy to ensure when folks are using their personal forks (think of it as an analog to personal email). Our policy right now is to save all repos for this reason. Additionally, I think it makes sense that work spent towards 18F projects gets associated with the 18F org; having contributions move through 18F also makes them easier to track.

I don't think forking to the 18F org implies we'll maintain the repo (and encourage frequent use of the "unmaintained" Github tag!)